.eslintignore

```diff
@@ -1 +1,7 @@
-node_modules
\ No newline at end of file
+/dist/**/*
+/docs/**/*
+/scripts/**/*
+/ts*/**/*
+/types/**/*
+.eslintrc.cjs
+rollup.config.js
```

Comment:
Let's remove folders from this list that aren't part of the repository if that's ok.